index.js

```diff
@@ -6,20 +6,50 @@ import { quickSort } from './utils/quickSort.js';
 import { menuConfig } from './dataStore/menuConfig.js';
 
 const tableContainer = document.getElementById('table');
+let countriesLenght = Object.keys(countries).length;
+const TOTAL_AMOUNT = countriesLenght;
 
+const callbacksObject = {
+  sortCallback: (dataKey, isAsc) => {
+    const isDataKey = Boolean(dataKey);
+    if (!isDataKey) {
+      return countriesTable.render(countries, TOTAL_AMOUNT);
+    }
+    isAsc
+      ? countriesTable.render(quickSort(countries, dataKey), TOTAL_AMOUNT)
+      : countriesTable.render(
+          quickSort(countries, dataKey).reverse(),
+          TOTAL_AMOUNT
+        );
+  },
+  paginationCallback: (currentPage, amount) => {
+    const isArgument = Boolean(currentPage);
+    if (!isArgument) {
+      return countriesTable.render(countries, TOTAL_AMOUNT);
```

Comment:
If you do not have the current page, you'll return a full dataset instead of split by pagination as required
I'm not sure that makes sense to have this check here, in which case we are able to get an empty current page? We always have at least `1 page`